test/integration/change-streams/change_stream.test.js

```diff
@@ -187,6 +187,77 @@ describe('Change Streams', function () {
   });
   afterEach(async () => await mock.cleanup());
 
+  context('ChangeStreamCursor options', function () {
+    let client, db, collection;
+
+    beforeEach(async function () {
+      client = await this.configuration.newClient().connect();
+      db = client.db('db');
+      collection = db.collection('collection');
+    });
+
+    afterEach(async function () {
+      await client.close();
+      client = undefined;
+      db = undefined;
+      collection = undefined;
+    });
+
+    context('fullDocument', () => {
+      it('sets fullDocument to `undefined` if no value is passed', function () {
```

Comment:
the test case says it sets fullDocument to `undefined`, but the assertion is for the property not being set at all on the object. Should it say "does not set the fullDocument property if no value is passed"?